src/db/guitar/chords/F/major.js

```diff
@@ -10,7 +10,7 @@ export default {
     },
     {
       frets: 'xx3211',
-      fingers: 'xx3211',
+      fingers: '003211',
```

Comment:
@emlyn nice fix. @tombatossals, can you incorporate this update so that the test on the master branch code can succeed? I did verify that it does work properly after this change is in place.